templates/repo/issue/filters.tmpl

```diff
@@ -18,18 +18,39 @@
 						<i class="icon">{{svg "octicon-search" 16}}</i>
 						<input type="text" placeholder="{{ctx.Locale.Tr "repo.issues.filter_label"}}">
 					</div>
+					<div class="ui checkbox compact archived-label-filter">
+						<input name="archived" type="checkbox"
```

Comment:
Hmm…
This checkbox is currently completely unaccessible.
I see, here's another candidate for where we need to improve the accessibility offered by Fomantic…

EDIT: Okay, surprisingly, there is one weird workaround to make it "accessible": Click on the label name field, then you can tab into the checkbox…
But I would not call that "accessible".

Reply:
the same